src/LaravelDataTables/Providers/ServiceProvider.php

```diff
@@ -1,11 +1,17 @@
 <?php namespace SevenD\LaravelDataTables\Providers;
 
+use App\Mail\DataTablesCsvEmail;
+use Illuminate\Support\Facades\Mail;
 use Illuminate\Support\ServiceProvider as LaravelServiceProvider;
+use Opis\Closure\SerializableClosure;
```

Comment:
Are we using this import?

Reply:
No. It's a leftover of unsuccessful attempts of using third party pack to serialize DataTable or Config classes, in order to use Jobs. Will remove it on next update.